### PR TITLE
SWITCHYARD-2942 - reacting to runtime version for JAXB transformer

### PR DESCRIPTION
**Pull request by bfitzpat**
creation. We add the new fields if it's 2.1+ and don't if it's < 2.1

### CODE REVIEW

File: eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java [hunk 1]
```diff
@@ -254,5 +270,15 @@ private boolean removePackages() {
             return changed;
         }
 
+        private boolean showNewControls() {
```
Comment (apupier) [inline eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java]: I would prefer to name the method isVersion21OrHigher. In few days, these controls won't be really new anymore :-)

File: eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java [hunk 2]
```diff
@@ -254,5 +270,15 @@ private boolean removePackages() {
...
+            boolean isVersion21OrHigher = true;
```
Comment (apupier) [inline eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java]: I would prefer to not use this intermediate variable and return !(version.startsWith("2.0") || version.startsWith("1")) directly.
or keeping the if/else clerarer
if (version.startsWith("2.0") || version.startsWith("1")) {
               return false;
          } else {
return true;
}

Comment (bfitzpat) [inline eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java]: +1 I'll change this

File: eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java (hunk 1 above)
Comment (bfitzpat) [inline eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java]: +1 I'll change this

**Comment by apupier** [comment]
+1

**Pull request merged**